portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var currSlideIndex = 0;
+var slides = document.querySelectorAll(".slide");
+showSlide(currSlideIndex);
```

Comment:
JS coding stylr - better to use either let or const, nor var

Reply:
done :)